openmicroscopy/openmicroscopy
# CLI import: expose more advanced Java import arguments

While reviewing 5.4.8-rc1, we realized that some of the advanced import arguments used for very fast CLI import including `--no-stats-info`, `--no-thumbnails` are not exposed in the Python arguments i.e. not displayed when calling `bin/omero import -h`.

This effectively prevents bulk configuration files to include these import arguments i.e. meaning skipping these steps need to be executed as `bin/omero import --bulk bulk.yml --skip <argument>` at the moment.

This PR adds these Java arguments to the `import` argument parsing as well as unit tests verifying these keys are recognized and passed to the Java command.

To test this PR:
- ~check that `bin/omero import -h`  includes the new options~
- create a minimal bulk configuration file similar to `components/tools/OmeroPy/test/unit/clitest/bulk_import/test_javaargs/bulk.yml` and check the import effectively skips thumbnail generation, min/max calculatin and upgrade check

/cc @dominikl
